### PR TITLE
Audit Fixes: Uniswap DTL

### PR DESCRIPTION
**Pull request by 0xJem**

### CODE REVIEW

File: src/callbacks/liquidity/BaseDTL.sol [hunk 1]
```diff
@@ -34,24 +40,27 @@ abstract contract BaseDirectToLiquidity is BaseCallback {
...
-    /// @param      proceedsUtilisationPercent  The percentage of the proceeds to deposit into the pool, in basis points (1% = 100)
-    /// @param      vestingStart                The start of the vesting period for the LP tokens (0 if disabled)
-    /// @param      vestingExpiry               The end of the vesting period for the LP tokens (0 if disabled)
-    /// @param      linearVestingModule         The LinearVesting module for the LP tokens (only set if linear vesting is enabled)
-    /// @param      active                      Whether the lot is active
-    /// @param      implParams                  The implementation-specific parameters
+    /// @param      recipient           Recipient of the LP tokens
+    /// @param      lotCapacity         Capacity of the lot
+    /// @param      lotCuratorPayout    Maximum curator payout of the lot
+    /// @param      poolPercent         Percentage of the proceeds to allocate to the pool, in basis points (1% = 100). The remainder will be sent to the `recipient`.
```
Comment (0xJem) [inline src/callbacks/liquidity/BaseDTL.sol:53]: > The remainder will be sent to the `recipient`.

This is inaccurate. It's sent to the seller currently.

In the Baseline DTL, it's sent to the specified recipient. I suggest we amend this to the recipient, in order to be consistent.

Comment (Oighty) [reply src/callbacks/liquidity/BaseDTL.sol:53]: Agree

**Comment by Oighty** [comment]
@0xJem looks like a lint error on the CI. Can you fix before we hand off?

**Comment by 0xJem** [comment]
CI build is failing as the CI runner cannot checkout the Baseline repo

**Pull request merged**